### PR TITLE
Generate transaction

### PR DESCRIPTION
**Pull request by TheEyePatch**
This PR is for generating the user_transaction model that joins the broker_stock model and the buyer model. This also generates the credentials that encrypts the secret keys from the API. The data from the API is then used to populate the stock model in the seeds.rb.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -37,3 +37,5 @@ group :test do
   gem 'rubocop-rails', require: false
   gem 'rubocop-rspec', require: false
 end
+
+gem 'iex-ruby-client'
```
Comment (analyn-cajocson) [inline Gemfile]: can you put it on the upper part of the Gemfile?

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,18 @@
...
+    endpoint: 'https://cloud.iexapis.com/v1'
+)
+symbols = client.ref_data_symbols()
+
+symbols.slice(0,20).each do |symbol|
+    quote = client.quote(symbol.symbol)
+    company = client.company(symbol.symbol)
+
+    Stock.create(symbol: symbol.symbol, price: quote.latest_price, company_name: company.company_name)
+end
```
Comment (analyn-cajocson) [inline db/seeds.rb]: add new line here

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (TheEyePatch) [reply Gemfile]: resposition gem 'iex-ruby-client'

File: db/seeds.rb (hunk 2 above)
Comment (TheEyePatch) [reply db/seeds.rb]: added new line in seeds.rb

File: Gemfile [hunk 3]
```diff
@@ -15,6 +15,7 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'devise'
 gem 'hamlit-rails'
+gem 'iex-ruby-client'
```
Comment (TheEyePatch) [inline Gemfile:18]: reposition gem 'iex-ruby-client'

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,19 @@
...
+
```
Comment (TheEyePatch) [inline db/seeds.rb:23]: added newline on seeds.rb

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**